cupy/cuda/runtime.pyx

```diff
@@ -187,8 +187,8 @@ cdef extern from 'cupy_cuda.h' nogil:
     int cudaDevAttrComputeCapabilityMajor
     int cudaDevAttrComputeCapabilityMinor
 
-_is_hip_environment = hip_environment
-is_hip = hip_environment
+_is_hip_environment = hip_environment  # for runtime being cimport'd
+is_hip = hip_environment  # for runtime being import'd
```

Comment:
I added short comments here because in a previous attempt (ad21969) with `is_hip` cpdef'd, I [got an `AttributeError`](https://travis-ci.org/cupy/cupy/builds/650305129#L361).